packages/common/src/utils/timeFrames.ts

```diff
@@ -280,13 +280,13 @@ export const timeFrameConfigs: Record<TimeFrames, TimeFrameConfig> = {
         getLabel: () => 'Day',
         getDimensionType: () => DimensionType.DATE,
         getSql: getSqlForTruncatedDate,
-        getAxisMinInterval: () => null,
+        getAxisMinInterval: () => 86400000,
     },
     WEEK: {
         getLabel: () => 'Week',
         getDimensionType: () => DimensionType.DATE,
         getSql: getSqlForTruncatedDate,
-        getAxisMinInterval: () => null,
+        getAxisMinInterval: () => 86400000,
```

Comment:
I believe this fixes the bug but why 🤔 can you add a comment in the code as well?

Reply:
I believe forcing the minimum interval to be 1 day will prevent echarts from showing nice timelines. Waiting for PR env to be ready to confirm this.

Reply:
yeah, the same applies to all other axis intervals, 86400000 means 1 day in milliseconds. @ZeRego originally implemented this, so he probably knows what kind of dark magic  this is :D

Reply:
https://github.com/lightdash/lightdash/pull/3346#discussion_r984590514

Reply:
:chefkiss:  [](edited-with-Axolo)

Reply:
In production:
![Screenshot 2022-11-08 at 13 45 34](https://user-images.githubusercontent.com/9117144/200581064-e9cd5c19-d952-456c-abaa-430d9928009a.png)

This PR:
![Screenshot 2022-11-08 at 13 39 20](https://user-images.githubusercontent.com/9117144/200581054-dda7c5e0-a5e6-4981-a1be-e8cec8db581f.png)

The problem with setting this value is that we force the axis points and lose the nice labels for the months and days. 
This is not a problem when dealing with large time intervals like year and quarter but uglier when dealing with day and week.

Reply:
Nice catch, let me have a look [](edited-with-Axolo)

Reply:
@TuringLovesDeathMetal would this work for you? is there a problem if we still show the `ticks`?